docs/modules/ROOT/pages/explanation/decisions/zone-naming.adoc

```diff
@@ -0,0 +1,34 @@
+= Zone naming scheme
+
+Problem::
+{product} will consist of several clusters we refer to as zones.
+We need a common understanding how we name them.
+
+Alternatives::
+* Using a randomly generated identifier like `rapid-sound-0933` or `rapid-sound` (`rapid-sound.appuio.cloud`).
+  We have tried that already with https://syn.tools[Project Syn].
+  This is way better than for example a random string of characters or a UUID.
+  From the perspective of daily operation, we have learned that this was still too much mental load even for a small number of items.
+* Use top level domain to encode country locality (`<zone>.appuio.ch`, `<zone>.appuio.de`).
+  This does not work as not every country has a country specific top level domain (for example the United States of America).
```

Comment:
The only downside of having everything on one zone is, that if we break DNS somehow, this can have an effect to all our appuio clusters. Would not change it, but might be we should document this.